include/zephyr/bluetooth/conn.h

```diff
@@ -416,6 +416,73 @@ struct bt_conn_le_tx_power {
 	int8_t max_level;
 };
 
+/** LE Transmit Power Reporting Structure */
+struct bt_conn_le_tx_power_report {
+
+	/** 0x00: local tx power changed.
+	 *  0x01: remote tx power changed.
+	 *  0x02: HCI_LE_Read_Remote_Transmit_Power_Level command completed.
+	 */
+	uint8_t reason;
+
+	/** 1M, 2M, Coded S2 or Coded S8 */
+	uint8_t phy;
+
+	/** Transmit power level */
+	int8_t tx_power_level;
+
+	/** 0: Transmit power level is at minimum level.
+	 *  1: Transmit power level is at maximum level.
+	 */
+	int8_t tx_power_level_flag;
+
+	/** Change in transmit power level */
+	int8_t delta;
+};
+
+enum bt_conn_le_path_loss_zone {
+	BT_CONN_LE_PATH_LOSS_ZONE_LOW = 0x00,
+	BT_CONN_LE_PATH_LOSS_ZONE_MIDDLE = 0x01,
+	BT_CONN_LE_PATH_LOSS_ZONE_HIGH = 0x02,
+};
+
+/** LE Path Loss Report Structure */
+struct bt_conn_le_path_loss_report {
+
+	/** Current path loss in dBm unit.
+	 * 0xFF: unavailable
+	 */
+	int8_t current_path_loss;
+
+	/** 0x00: entered to low zone.
+	 *  0x01: entered to middle zone.
+	 *  0x02: entered to high zone.
+	 */
+	enum bt_conn_le_path_loss_zone zone_entered;
+};
+
+/** LE Set Path Loss Reporting Parameters Structure */
+struct bt_conn_le_path_loss_reporting_param {
+	/** High threshold for the path loss in dBm unit.
+	 * 0xFF: High threshold unused.
+	 */
+	int8_t high_threshold;
+
+	/** High hysteresis for the high threshold in dBm unit. */
+	int8_t high_hysteresis;
+
+	/** Low threshold for the path loss in dBm unit. */
```

Comment:
No. it allows to use 2 zones instead of 3 when high threshold is not used (0xFF). 
![image](https://user-images.githubusercontent.com/53769824/181771410-dbc0f14a-163d-41a0-a2e9-88906ec816c7.png)